src/main/java/core/basesyntax/service/Return.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitBox;
+import java.util.Map;
+
+public class Return extends Operation {
+    public Return(OperationType type, FruitBox fruit) {
+        super(type, fruit);
```

Comment:
Sorry but I think you choose not the best name of the class - word return is used in so many different ways in this code:). Not critical comment at all